### ISSUE TITLE
Missing aria-label from v-select

### ISSUE THREAD

**Issue by tpamsler** [vuetifyjs/vuetify#22629]
The aria-label is missing from the v-select <div> with role="combobox". SiteImprove flags this as an "A" issue.
 
Vuetify : v3.11.8
Vue : v3.5.28
 <img width="1911" height="723" alt="Image" src="https://github.com/user-attachments/assets/85f89fd7-0b52-44df-9b67-1843faaaa64a" />

**Comment by J-Sek** [comment]
`aria-label` lands on the `<input />` inside (after #21903), so the `role` appears in 2 places.
Does it actually affect assistive technologies?

**Comment by tpamsler** [comment]
Good point, @J-Sek. I just tested the v-select using Mac OS X VoiceOver and keyboard navigation, and I didn’t encounter any issues while navigating the component or missing announcements.

**Comment by prarie** [comment]
I found this issue because I was having trouble with Playwright's automated locators not resolving to the expected `getByRole('combobox', { name: 'Region' })` nor does the locator actually work when resolving the element if it's "hand coded". 

Playwright uses aria selectors if at all possible (and most things work great with Vuetify, which is awesome). While I'm not sure, I suspect the root cause is related to this report. We're running Vuetify `3.12.5`.
